apps/dashboard/app/layout.tsx

```diff
@@ -37,8 +38,16 @@ export default function RootLayout({
               },
             }}
           />
-          <header className="header">Header</header>
-          {children}
+          <Header isLogged />
+          <Box
+            sx={{
+              width: "100%",
+              maxWidth: "1200px",
+              margin: "0 auto",
+            }}
+          >
+            {children}
+          </Box>
```

Comment:
```suggestion
          <Box
            sx={{
              width: "100%",
              maxWidth: "1200px",
              margin: "0 auto",
              padding: "24px"
            }}
          >
            {children}
          </Box>
```

Para resolver o problema do mobile mencionado no outro comentário.